pages/glossary/programmatically-determined-context.md

```diff
@@ -8,11 +8,12 @@ The programmatically determined context of a piece of information (_e.g._ an ele
 It includes:
 
 - the element's [accessible name][];
-- elements that are referred by the `aria-describedby` attribute; 
+- elements that are referred by the `aria-describedby` attribute;
 - the `title` attribute;
 - ancestor `label` elements;
 - `label` elements referring the current element with a `for` attribute;
 - table headers associated with the cell containing the information;
+- the header that precedes the element;
```

Comment:
It's probably more than just the one preceding the element, more like the full hierarchy of headers going to it.
```html
<h1>Headers</h1>
<h2>Levels</h2>
<h3>Possible levels</h3>
<h3>Hierachy</h3>
<span id="target">Do not skip levels!</span>
```
The header context of the `span` is the `h1`, the `h2` and the second `h3`.

Reply:
@carlosapaduarte do you think this should pass or fail?
```html
<h1>Hamlet</h1>
  <h2>Act 1</h2>
    <h3>Scene 1</h3>
      <a href="…">HTML</a> <a href="…">PDF</a> <a href="…">DOCX</a>
    <h3>Scene 2</h3>
      <a href="…">HTML</a> <a href="…">PDF</a> <a href="…">DOCX</a>
  <h2>Act 2</h2>
    <h3>Scene 1</h3>
      <a href="…">HTML</a> <a href="…">PDF</a> <a href="…">DOCX</a>
    <h3>Scene 2</h3>
      <a href="…">HTML</a> <a href="…">PDF</a> <a href="…">DOCX</a>
```

I think this passes because the full headers hierarchy is providing context to understand what the links mean.

Reply:
@Jym77 yes, I agree that passes because of the full hierarchy. I'll update accordingly